exercises/nucleotide-count/src/test/java/NucleotideCounterTest.java

```diff
@@ -106,4 +63,12 @@ public void testCountsAllNucleotides() {
                 hasEntry('T', 21)
         ));
     }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testDnaStringHasInvalidNucleotides() {
+        expectedException.expect(NullPointerException.class);
```

Comment:
Instead of leaving the program to throw a `NullPointerException`, what about using an `IllegalArgumentException` or something similar - it could even be a custom exception if you wanted!

Reply:
That looks reasonable to me :slightly_smiling_face: